blocks/hooks/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * WordPress dependencies
  */
-import createHooks from '@wordpress/hooks';
```

Comment:
I guess this should can still be imported from '@wordpress/hooks'?

Reply:
OR, why did you expose the "createHooks" in `@wpcore/hooks`?

Reply:
Yes, makes sense. I will update both.

Reply:
The only downside of importing `@wordpress/hooks` directly is that it will get bundled twice.